### DISCUSSION TITLE
Support für SubLineTradeLineItems

### DISCUSSION THREAD

**Discussion by stackoverwolf** [stephanstapel/ZUGFeRD-csharp#403] [Q&A]
Bei einigen Rechnungen müssen die Positionen gruppiert werden, dies scheint mit SubLineTradeLineItems möglich zu sein.
Beispiele bei denen die Gruppierung nötig ist:
- Sammelrechnung
- Zusammenfassung von verschiedenen Positionen zu einer Hauptgruppe

Ist der Support für SubLineTradeLineItems bereits geplant?

**Comment by stephanstapel** [comment]
Interessanter Punkt, @stackoverwolf : hast Du weitere Informationen dazu?

**Reply by stackoverwolf** [reply]
Ich konnte dazu bisher folgendes Finden
- https://www.e-rechnung-bund.de/faq/xrechnung/
  - unter Extension XRechnung stehen mehrere Informationen
- Oder in der XRechnung Dokumentation https://github.com/stephanstapel/ZUGFeRD-csharp/blob/master/documentation/xRechnung/XRechnung%203.0.1/XRechnung-v3.0.1.pdf
  - unter Kapitel 16.1. Gruppe SUB INVOICE LINE
  - hier wird auch beschrieben, dass die "Sub Invoice Line" selbst auch wieder "Sub Invoice Lines" besitzen kann
  - teilweise sind die Properties auch identisch mit den "Invoice Lines"

**Comment by HenriSHS** [comment]
#430
